rules/windows/process_creation/proc_creation_win_lolbin_agentexecutor.yml

```diff
@@ -0,0 +1,29 @@
+title: Suspicious AgentExecutor PowerShell Execution
+id: 7efd2c8d-8b18-45b7-947d-adfe9ed04f61
+status: experimental
+description: Detects AgentExecutor.exe execution of binary named powershell when the default ps path is absent from the command line. AgentExecutor will execute any binary named powershell.exe located in the $PSFolder path argument. AgentExecutor is included as part of Intune Managed Devices for Windows 10. 
+references:
+    - https://twitter.com/lefterispan/status/1286259016436514816
+    - https://lolbas-project.github.io/lolbas/OtherMSBinaries/Agentexecutor/
+    - https://docs.microsoft.com/en-us/mem/intune/apps/intune-management-extension
+tags:
+    - attack.defense_evasion
+    - attack.t1218
+date: 2022/07/31
+author: memory-shards
+logsource: 
+    category: process_creation
+    product: windows
+detection:
+    selection1:
+        Image: 'AgentExecutor.exe'
+    selection2: 
+        CommandLine|contains: '-powershell'
+    filter:
+        CommandLine|contains:
```

Comment:
Filter do not seen usefull
From lolbas :
`AgentExecutor.exe -powershell "c:\temp\malicious.ps1" "c:\temp\test.log" "c:\temp\test1.log" "c:\temp\test2.log" 60000 "C:\Windows\SysWOW64\WindowsPowerShell\v1.0" 0 1`

Reply:
Resolving the discussions because I reworked the rule.